src/browser/commands/assert-view/index.js

```diff
@@ -155,7 +155,23 @@ module.exports.default = browser => {
         testplaneCtx.assertViewResults.add({ stateName: state, refImg: refImg });
     };
 
-    session.addCommand("assertView", async function (state, selectors, opts = {}) {
+    session.addCommand("assertView", async function (state, selectorsOrOpts, opts = {}) {
+        let selectors;
```

Comment:
as for me, the code turned out to be a little be dirty here. Why not just create two functions: `assertViewBySelector` and `assertViewByViewport`?

The code would look something like this:
```
const waitSelectorsForExist = (...) => {
  await Promise.map([].concat(selectors), async selector => {...});
};

const assertViewBySelector = (...) => {
  await waitSelectorsForExist(...);

  return assertView(state, selectors, opts);
};

const assertViewByViewport = (...) => {
  // prepare opts
  
  return assertView(state, selectors, opts);
};

const shouldAssertViewport = (...) => {
  return typeof selectorsOrOpts === "string" || _.isArray(selectorsOrOpts);
};

session.addCommand("assertView", async function (state, selectorsOrOpts, opts = {}) {
  return shouldAssertViewport ? assertViewByViewport(...) : assertViewBySelector(...);
});
```

What do you think? To my mind this code is much easier to read.